tests/python/test_bit_operations.py

```diff
@@ -13,8 +13,14 @@ def test_bit_shl():
     def shl(a: ti.i32, b: ti.i32) -> ti.i32:
         return a << b
 
+    @ti.kernel
+    def shl_assign(a: ti.i32, b: ti.i32) -> ti.i32:
+        c = a
+        c <<= b
+        return c
+
     for i in range(8):
-        assert shl(3, i) == 3 * 2**i
+        assert shl(3, i) == shl_assign(3, i) == 3 * 2**i
```

Comment:
`3 <<= 1` gives the same result regardless of whether it's "signed shift" or "unsigned shift". Shall we test with negative number or positive number large enough(`01xxxxxx`) to influence the sign-bit after shift?

Reply:
`left shift` gives the same result anyway. Only `right shift` will be influenced by the sign bit.

Reply:
IIRC shl and sal are the same, and they do not preserve the signed bit.